packages/algolia/src/client.ts

```diff
@@ -1,13 +1,22 @@
 import { SearchIndex } from 'algoliasearch';
 import { SearchOptions, SearchResponse } from '@algolia/client-search';
-import { ResearchOutputResponse, UserResponse } from '@asap-hub/model';
+import {
+  ExternalAuthorResponse,
+  LabResponse,
+  ResearchOutputResponse,
+  UserResponse,
+} from '@asap-hub/model';
 
 export const RESEARCH_OUTPUT_ENTITY_TYPE = 'research-output';
 export const USER_ENTITY_TYPE = 'user';
+export const EXTERNAL_AUTHOR_ENTITY_TYPE = 'external-author';
+export const LAB_ENTITY_TYPE = 'Lab';
```

Comment:
if anything it should be
```suggestion
export const LAB_ENTITY_TYPE = 'lab';
```